release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -122,6 +122,26 @@ Release Team selection should happen in accordance with the [Release Team select
   - Kubernetes 1.16: Unlimited Breadsticks For All
   - Kubernetes 1.17: The Chillest Release - Capynetes
 
+### Mailing List Standards
+
+Important communications and discussions should be posted in the release team
+mailing list https://groups.google.com/forum/#!forum/kubernetes-release-team.
+
+This list is owned by SIG Release and as such the SIG Release chairs are owners
+of it.
+While SIG Release leads are owners of this list, the release team lead and
+release lead shadows will serve as managers of the list.
+
+The release team lead and release lead shadows should periodically go through
```

Comment:
Should this be ambiguously "periodically", or a more specific time related to a new cycle having started?

Reply:
It would be good to be more specific here.
One definite time to prune/update is once shadows are locked in and ideally _before_ sending new cycle invites or emails.

I've gotten a few comments in the past from people who were not selected for the next cycle but still got a (confusing) welcome email.

Reply:
Maybe something like:
 "After shadows have been selected, update the list by adding the new shadows and removing any previous members who did not stay on for the current release cycle."